layouts/_default/list.html

```diff
@@ -2,28 +2,30 @@
 {{ range .Pages }}
 <div class="box">
   <article class="media">
-    <!--
-    <div class="media-left">
-      <figure class="image is-64x64">
-        <img src="https://bulma.io/images/placeholders/128x128.png" alt="Image">
-      </figure>
-    </div>
-    -->
     <div class="media-content">
       <div class="content">
         <p>
-          <a href="{{.Permalink }}" class="title is-size-5"><strong>{{ .Title }}</strong></a>{{ with .Params.Author }} <small>@ {{ delimit . ", " }}</small>{{ end }}<br>
+          <a href="{{.Permalink }}" class="title is-size-5"><strong>{{ .Title }}</strong></a>
+          {{ with .Params.Author }} <small>@ {{ delimit . ", " }}</small>{{ end }}
+          <br>
           {{.Description}}
         </p>
       </div>
       <nav class="level is-mobile">
         <div class="level-left">
           {{- partial "icons.html" . -}}
         </div>
-        <div class="level-right">
-        </div>
       </nav>
     </div>
+    {{ $dir := "static/logos/" }}
+    {{ $img := lower (print .File.BaseFileName ".png") }}
+    {{ if (where (readDir $dir) "Name" $img) }}
```

Comment:
I think you want use `with` here: https://gohugo.io/functions/with/

Reply:
Sorry, I made a lot of attempts, but I not figured out how to use *with* in this case (remember, I am not an expert with Hugo)